src/main/java/picard/util/LiftoverUtils.java

```diff
@@ -312,14 +329,13 @@ protected static void leftAlignVariant(final VariantContextBuilder builder, fina
         int theStart = start;
         int theEnd = end;
 
-        final String refString = StringUtil.bytesToString(referenceSequence.getBases(), start - 1, end - start + 1);
         //make sure that referenceAllele matches reference
-        final Allele refAllele = alleles.stream().filter(Allele::isReference).findAny().orElseThrow(() -> new RuntimeException("How did that happen? No reference allele?"));
-
-        if (!refString.equalsIgnoreCase(refAllele.getBaseString())) {
+        final String refString = StringUtil.bytesToString(referenceSequence.getBases(), start - 1, end - start + 1);
+        if (!referenceAlleleMatchesReference(alleles, refString)) {
+            final Allele refAllele = alleles.stream().filter(Allele::isReference).findAny().orElse(null);
             throw new IllegalArgumentException(
                     String.format("Reference allele doesn't match reference: Allele= %s, at %s:%d-%d, ref=%s",
-                            refAllele.toString(), referenceSequence.getName(), start, end, refString));
+                            refAllele == null ? "null" : refAllele.toString(), referenceSequence.getName(), start, end, refString));
```

Comment:
this is odd...if the reference allele doesn't match the reference, how could you not have a reference allele?